docs/source/learn/examples/dimensionality.ipynb

```diff
@@ -0,0 +1,654 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# PyMC Dimensionality\n",
+    "PyMC provides a number of ways to specify the dimensionality of its distributions. In this document we will not provide an exhaustive explanation but rather an overview and current best practices. \n",
+    "\n",
+    "## Glossary\n",
+    "In this document we'll be using the term dimensionality to refer to the idea of dimensions. Each of the terms below has a specific\n",
+    "semantic and computational definition in PyMC.  While we share them here they will make much more sense when viewed in the examples below.\n",
+    "\n",
+    "+ *Implied dimensions* → dimensionality that follows from inputs to the RV\n",
+    "+ *Support dimensions* → dimensions you can NEVER get rid of\n",
+    "+ *`ndim_support`* → smallest shape that can result from a random draw. This is a fixed attribute in the distribution definition\n",
+    "+ *Shape* → final resulting tensor shape\n",
+    "+ *Size* → shape minus the support dimensions\n",
+    "+ *Dims* → An array of dimension names\n",
+    "+ *Coords* → A named array that can be used to specify dimensionality\n",
```

Comment:
```suggestion
    "+ *Coords* → A dictionary mapping dimension names to coordinate values\n",
```
Related, this is how I defined [it in the McBackend protobuf](https://github.com/michaelosthege/mcbackend/blob/2706f8570e395bdcc006111911f6716d73ecc5ee/protobufs/meta.proto#L40-L46):
```proto
// A named array that labels a dimension.
message Coordinate {
    // Name of the dimension these labels belong to.
    string name = 1;
    // Coordinate values.
    npproto.ndarray values = 2;
}
```